sass/color/_contrast.scss

```diff
@@ -138,13 +140,13 @@ $WCAG-CONTRAST: (
   $color: map-get($contrast, 'color');
```

Comment:
should  this be `map.get`?

Reply:
oh, I haven't updated the color module yet. So yes, but I'm doing that in another branch. Oops, I'll note that for Stacy & Jonny as well…